packages/components/src/templates/next/render/renderComponentPreviewText.ts

```diff
@@ -95,6 +95,8 @@ export function renderComponentPreviewText({
       return component.title
     case "map":
       return component.title || "Map embed"
+    case "logocloud":
+      return component.title || "Logo cloud"
```

Comment:
nit - since title is required, shouldn't need to fallback to "Logo cloud"

Reply:
kek i was copying HAHA thakns for catch, fixed!